app/views/proprietor/accounts/_add_data_for_cross_tenant_search.html.erb

```diff
@@ -0,0 +1,25 @@
+
+
+<p>
+  <%= f.check_box :shared_search, {checked: account.shared_search_tenant?, name: "account[settings][shared_search]", id: "account_settings_shared_search"}, true , false %>
+  <%= f.label "Search Only(please tick checkbox select tenants)" %>
+</p>
+<br />
+
+<%= f.label :tenant_list %>
+
+<p>
+  <%  @account.children.each do |tenant| %>
+    <%= check_box_tag "account[settings][tenant_list][]", tenant.tenant, @account.children.include?(tenant) %>
```

Comment:
This checkbox tag could go in a helper as well

Reply:
Ok noted